doc/fluid/design/concepts/cpp_data_feeding.md

```diff
@@ -169,3 +173,30 @@ Two important considerations for these programs are as follows:
 1. The multiple\_reader is the batch\_reader's underlying reader, and the batch\_reader is the double\_buffer\_reader's underlying reader. `read_op`, `has_next_op` and other reader related ops will only invoke the top-most reader. In this case, it's the double\_buffer\_reader.
 
 2. All readers exist in both `startup_program` and `main_program`. And they are persistable.
+
+### Simplify Configuration by MultiPassReader
+
+The  Program configuration mentioned above is somehow complicated. Users need to be very similar to concepts of Program and Block to prevent making mistakes in their code. To make the usage of C++ readers more friendly to beginning users, we introduce `MultiPassReader`.
```

Comment:
The  Program -> The Program (please change two spaces to one space).
similar -> familiar

Reply:
Done. Thanks!